packages/bubble-dev/start-preset/src/build.ts

```diff
@@ -87,36 +88,54 @@ export const buildReactNative = async (dir: string): Promise<StartPlugin<{}, {}>
 
 export const buildNode = async (dir: string): Promise<StartPlugin<{}, {}>> => {
   const { babelConfigNodeBuild } = await import('@bubble-dev/babel-config')
-
-  return sequence(
-    env({ BABEL_ENV: 'production' }),
-    find([
+  const { default: globby } = await import('globby')
+  const allFiles = await globby(
+    [
       `${dir}/src/**/*.{js,jsx,ts,tsx}`,
       `!${dir}/src/**/*.{native,ios,android}.{js,jsx,ts,tsx}`,
       `!${dir}/src/**/*.d.ts`,
-    ]),
-    read,
-    babel(babelConfigNodeBuild),
-    rename((file) => file.replace(/\.(ts|tsx|jsx)$/, '.js')),
-    write(`${dir}/build/node/`),
-    find(`${dir}/src/**/*.json`),
-    copy(`${dir}/build/node/`),
-    plugin('test', () => async () => {
-      const path = await import('path')
-      const { access } = await import('pifs')
-      const fullPath = path.resolve(`${dir}/build/node/index.js`)
-      let hasFile = false
-
-      try {
-        await access(fullPath)
-        hasFile = true
-      } catch {}
-
-      if (hasFile) {
-        await import(fullPath)
-      }
-    })
+    ],
+    {
+      ignore: ['node_modules/**'],
+      deep: Infinity,
+      onlyFiles: true,
+    }
   )
+  const extRegExp = /\.(js|jsx|ts|tsx)$/
+  const nodeFiles = allFiles.filter((file) => {
```

Comment:
This filter grabs all the files and filters out the files with the same name as the files with `.node.(js|jsx|ts|tsx)` extension. If there are non, it will just use the main file.